tests/core/full_node/full_sync/test_full_sync.py

```diff
@@ -208,6 +209,46 @@ def fn3_is_not_syncing():
         await time_out_assert(180, node_height_exactly, True, full_node_1, 999)
         await time_out_assert(180, node_height_exactly, True, full_node_2, 999)
 
+    @pytest.mark.asyncio
+    async def test_sync_none_wp_response_backward_comp(self, three_nodes, default_1000_blocks):
```

Comment:
Is it possible to add a test for the failure case, where a peer does not support a capability?

Reply:
inline 220 the capabilities of node 3  are reset to include only base, we later check that it behaves as expected